### PR TITLE
maintain: const over let, consistency

### PR DESCRIPTION
**Pull request by kmdrGroch**
## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions.

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Comment by kmdrGroch** [comment]
I have updated some files to be consistency and more written in "perfect ES6 style".
You might also would like to add some linter or writing in typescript in the future, since right now you are starting to losing consistency in your files, use `var` or `let` where you don't need it. If you agree I can configure ESlint for you to match your style, or try rewriting code to typescript

File: lib/build.js [hunk 1]
```diff
@@ -47,18 +47,16 @@ const touchOverriddenVectorIconFiles = () => {
   // Return true when original file of |file| should be touched.
   const applyFileFilter = (file) => {
     // Only includes icon files.
-    const ext = path.extname(file)
-    if (ext !== '.icon') { return false }
-    return true
+    return path.extname(file) === true
```
Comment (spirinvladimir) [inline lib/build.js]: Looks like misprint..
```
return path.extname(file) === '.icon'
```

File: lib/build.js [hunk 2]
```diff
@@ -19,12 +19,12 @@ const touchOverriddenFiles = () => {
...
+  const sourceFiles = util.walkSync(chromiumSrcDir, applyFileFilter)
 
   // Touch original files by updating mtime.
   const chromiumSrcDirLen = chromiumSrcDir.length
   sourceFiles.forEach(chromiumSrcFile => {
-    var overriddenFile = path.join(config.srcDir, chromiumSrcFile.slice(chromiumSrcDirLen))
+    const overriddenFile = path.join(config.srcDir, chromiumSrcFile.slice(chromiumSrcDirLen))
     if (!fs.existsSync(overriddenFile)) {
       // Try to check that original file is in gen dir.
       overriddenFile = path.join(config.outputDir, 'gen', chromiumSrcFile.slice(chromiumSrcDirLen))
```
Comment (spirinvladimir) [inline lib/build.js]: assign to const ```overriddenFile```!

**Comment by kmdrGroch** [comment]
adapted @mshirlaw and @spirinvladimir suggestions.
Yea, it was a typo with that icon.

Reverted changes in python - though there shouldn't be any difference in the file functionality. Especially in that indent change in the last line. But agree - JS files are JS files.

File: lib/pullL10n.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ const config = require('../lib/config')
 const util = require('../lib/util')
 const {braveTopLevelPaths, ethereumRemoteClientPaths} = require('./l10nUtil')
 
+
```
Comment (petemill) [inline lib/pullL10n.js]: Please remove this extra line 😄

**Comment by kmdrGroch** [comment]
Applied @petemill's suggestion and merged master (please check out the second commit if I merged correctly)

**Comment by bsclifton** [comment]
Closing as stale - sorry we didn't revisit this. We've since moved some of the code over to `brave-core`

**Pull request closed without merging**